diff --git a/node_modules/react-native-vision-camera/lib/module/expo-plugin/withVisionCamera.js b/node_modules/react-native-vision-camera/lib/module/expo-plugin/withVisionCamera.js
index 405acd1..2db6dd8 100644
--- a/node_modules/react-native-vision-camera/lib/module/expo-plugin/withVisionCamera.js
+++ b/node_modules/react-native-vision-camera/lib/module/expo-plugin/withVisionCamera.js
@@ -22,7 +22,8 @@ const withCamera = (config, props = {}) => {
     config.ios.infoPlist.NSLocationWhenInUseUsageDescription = props.locationPermissionText ?? config.ios.infoPlist.NSLocationWhenInUseUsageDescription ?? LOCATION_USAGE;
   }
   const androidPermissions = ['android.permission.CAMERA'];
-  if (props.enableMicrophonePermission) androidPermissions.push('android.permission.RECORD_AUDIO');
+  // PATCHED BY PANHANDLER - NEVER ADD AUDIO PERMISSIONS
+  // if (props.enableMicrophonePermission) androidPermissions.push('android.permission.RECORD_AUDIO');
   if (props.enableLocation) androidPermissions.push('android.permission.ACCESS_FINE_LOCATION');
   if (props.enableLocation != null) {
     // set Podfile property to build location-related stuff
